lib/wallaby/browser.ex

```diff
@@ -199,15 +199,17 @@ defmodule Wallaby.Browser do
   Takes a screenshot of the current window.
   Screenshots are saved to a "screenshots" directory in the same directory the
   tests are run in.
+
+  Pass %{name: "some_name"} to specify the file name. Defaults to a timestamp.
   """
   @spec take_screenshot(parent) :: parent
 
-  def take_screenshot(%{driver: driver} = screenshotable) do
+  def take_screenshot(%{driver: driver} = screenshotable, opts \\ %{}) do
```

Comment:
I know this is already merged, but what would you think about making opts a keyword list instead of a map and updating the typespec on line 205?

Reply:
@aaronrenner Sounds good to me. Will give it a go.